index.js

```diff
@@ -116,5 +116,18 @@ module.exports = {
           break;
     }
     return true;
+  },
+
+  /**
+   * Check if given date is past the current date
+   * @param date
+   * @param currentFormat
+   * @returns {boolean}
+   */
+   isDatePast: function ( date, currentFormat ) {
+     const currentDate = moment().format(currentFormat);
+     let isPast = moment(date).isBefore(currentDate);
```

Comment:
You can directly return 

`moment(date).isBefore(currentDate);`

as there is no use of using a new variable `isPast`

Reply:
Ok just did this one.